nilearn/plotting/displays.py

```diff
@@ -137,6 +140,35 @@ def draw_left_right(self, size, bg_color, **kwargs):
                 bbox=dict(boxstyle="square,pad=0", ec=bg_color, fc=bg_color),
                 **kwargs)
 
+    def draw_scale_bar(self, bg_color, scale_width=5, units='cm', **kwargs):
+        from mpl_toolkits.axes_grid1.anchored_artists import AnchoredSizeBar
+        ax = self.ax
+        fontproperties = kwargs.get('fontproperties', FontProperties())
```

Comment:
fontproperties -> font_properties (easier to read and PEP8 compliant).
Same with
frameon -> frame_on

Reply:
ax -> axis or axes depending on one axis or multplie